### PR TITLE
style: import front-end modules with full path

### PR DESCRIPTION
**Pull request by ChristophWurst**
Include .vue and .js in imports.

This is groundwork for the overdue eslint update in this repo.

### CODE REVIEW

File: src/tests/unit/components/NavigationMailbox.vue.spec.js [hunk 1]
```diff
@@ -22,8 +22,8 @@
 import {createLocalVue, shallowMount} from '@vue/test-utils'
 import Vuex from 'vuex'
 
-import NavigationMailbox from '../../../components/NavigationMailbox'
-import Nextcloud from '../../../mixins/Nextcloud'
+import NavigationMailbox from '../../../components/NavigationMailbox.js'
```
Comment (st3iny) [inline src/tests/unit/components/NavigationMailbox.vue.spec.js:25]:
```suggestion
import NavigationMailbox from '../../../components/NavigationMailbox.vue'
```

The linter complained about other tests as well.

Comment (ChristophWurst) [reply src/tests/unit/components/NavigationMailbox.vue.spec.js:25]: Works on my machine

Comment (st3iny) [reply src/tests/unit/components/NavigationMailbox.vue.spec.js:25]: Oops, I meant the js unit tests.

Does `npm run test:unit` work for you?

Comment (ChristophWurst) [reply src/tests/unit/components/NavigationMailbox.vue.spec.js:25]: Nope. I'm checking.

**Review by st3iny — APPROVED** [review]

**Comment by kesselb** [comment]
Damn, the rebase button does not autosquash :see_no_evil:

**Review by kesselb — APPROVED** [review]
Nice :+1:

**Pull request merged**